src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/DataBlockRequests.cs

```diff
@@ -0,0 +1,59 @@
+#nullable enable
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using FluentValidation;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Chart;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Requests;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+
+public record DataBlockCreateRequest
+{
+    [Required] public string Heading { get; init; } = string.Empty;
```

Comment:
I've been using the `required` _keyword_ for `...Request` classes, as my understanding was that the `[Required]` attribute was more of a code-first EF Core approach (based on other classes across the project).